### ISSUE TITLE
Support `\mancube` as in latex and related symbols

### ISSUE THREAD

**Issue by ice1000** [XmacsLabs/mogan#316]
![image](https://user-images.githubusercontent.com/16398479/191057612-38c79599-7e4d-4189-a271-63720f8d4733.png)

These are soooooooooooooooooo coooooooooooooool definitely deserve a place

**Comment by YiqiXu** [comment]
Do you know if there are Unicode-numbers for these symbols?

**Comment by ice1000** [comment]
I believe that these symbols are drawn directly

**Comment by darcy-shen** [comment]
@jingkaimori this issue should be solved by virtual font

**Comment by jingkaimori** [comment]
I don't think so, manfmt package relies on Knuth's `manual` font family, which should be added into encoding translator firstly.

**Comment by darcy-shen** [comment]
Well, let us make fonts related improvements in the upcoming 1.2.x. I have planned to design font plugins like language plugins.
